otel-collector-config.yaml

```
@@ -0,0 +1,15 @@
+      http:
+      grpc:
+
+exporters:
+  debug:
+    verbosity: detailed
+
+service:
+  pipelines:
+    traces:
+      receivers: [otlp]
+      exporters: [debug]
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)